suspect/fitting/singlet.py

```diff
@@ -207,29 +203,27 @@ def model_to_parameters(model_dict):
 
         # Construct lmfit Parameter input for each parameter.
         for name1, value1 in model_dict.items():
```

Comment:
It would be nice to rename these variables (name1, value1 etc.) to something a little more descriptive, see if you can come up with something. One thing I might suggest is handling the two phase parameters outside the loop and then looping over the peak dictionaries alone. Don't forget to work on a copy of the model if you do that.